com.github.ransome1.sleek.yml

```diff
@@ -11,6 +11,7 @@ finish-args:
   - --socket=x11
   - --share=network
   - --device=dri
+  - --filesystem=host
```

Comment:
why "host" isn't "home" enough?

Reply:
Because home doesn't include external hdd or network shares. And the use case here is that a user is suppose to choose by himself, where he wants to save or open his todo.txt file from.

Reply:
in case this is a blocker out of security reasons, I changed it to "home".